notebooks/1_BayesianWorkflow/1_Ins_BayesRefresher.ipynb

```diff
@@ -4,39 +4,50 @@
    "cell_type": "markdown",
```

Comment:
<p>Is likelihood notation flipped? On wikipedia its shown as L(theta|y)? Added a screenshot in comment below</p><p><br></p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/canyon289/bayesian-model-evaluation/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
The likelihood function notation is flipped compared with the probability notation. I like the probability notation because treats all terms of Bayesian theorem the same, and also I can avoid explaining the likelihood is not s probability but proportional to a probability.